resources/puppetlabs/puppetdb/bootstrap.cfg

```diff
@@ -13,11 +13,8 @@ puppetlabs.trapperkeeper.services.webrouting.webrouting-service/webrouting-servi
 puppetlabs.puppetdb.cli.services/puppetdb-service
 puppetlabs.puppetdb.http.command/puppetdb-command-service
 puppetlabs.puppetdb.metrics/metrics-service
-puppetlabs.puppetdb.dashboard/dashboard-service
-puppetlabs.puppetdb.admin/admin-service
 puppetlabs.puppetdb.mq-listener/message-listener-service
 puppetlabs.puppetdb.command/command-service
-puppetlabs.puppetdb.meta/metadata-service
```

Comment:
These probably need a corresponding PR to update them in PE so we don't end up having to do https://github.com/puppetlabs/pe-puppetdb-extensions/pull/78 again